src/components/molecules/SearchResultTile/index.tsx

```diff
@@ -27,9 +31,11 @@ const SearchResultTile: FC<ProblemSummaryProps> = ({
             tags.map((tagText: string) => <Tag text={tagText} key={tagText} />)}
         </S.TagWrapper>
       </S.DetailsWrapper>
-      <Button isPrimary={false} isHidden>
-        Zobacz treść
-      </Button>
+      <Link to={`/problems/${id}`}>
+        <Button isPrimary={false} isHidden>
+          Zobacz treść
+        </Button>
+      </Link>
```

Comment:
I think ideal would be that we have Link from react router that is styled as component, as having button tag inside of a link tag seems a bit odd. ;)